flow/core/experiment.py

```diff
@@ -241,5 +243,8 @@ def rl_actions(*_):
                              'fact_vehicle_trace/date={0}/partition_name={1}/{1}.csv'.format(cur_date, source_id),
                              trajectory_table_path,
                              {'network': metadata['network'][0], 'is_baseline': metadata['is_baseline'][0]})
+                upload_to_s3('circles.data.pipeline',
+                             'time_space_diagram/date={0}/partition_name={1}/{1}.csv'.format(cur_date, source_id),
```

Comment:
the S3 key for the diagram should have `.png` at the end here

Reply:
ah thanks!